src/manifestoo/main.py

```diff
@@ -253,6 +254,36 @@ def list_depends(
     )
 
 
+@app.command()
+def list_codepends(
+    ctx: typer.Context,
+    separator: Optional[str] = typer.Option(
+        None,
+        help="Separator character to use (by default, print one item per line).",
+    ),
+    transitive: bool = typer.Option(
+        True,
+        "--transitive",
+        help="Print all transitive co-dependencies.",
+    ),
+    include_selected: bool = typer.Option(
+        True,
+        "--include-selected",
+        help="Print the selected addons along with their co-dependencies"
+        "(the set of addons that depends on the set of selected addons).",
+    ),
+) -> None:
+    """Print the co-dependencies of selected addons."""
```

Comment:
Le's put the explanation about what a co-dependency is here rather than in the --include-selected description ?